6-7-8/script.js

```diff
@@ -0,0 +1,84 @@
+"use strict"
+
+let tasks;
+!localStorage.tasks ? tasks = [] : tasks = JSON.parse(localStorage.getItem('tasks'));
+
+let todoItemElems = [];
+
+class createTask {
+    constructor(about) {
+        this.about = about;
+        this.datetask = creationDate();
+    }
+}
+
+const createTemplate = (task, event) => {
+    return `
+    <div class = "todo-item ${task.isDone ? 'checked' : ''}">
+        
+        <div class = "buttons">
+            <input onclick = "completeTask(${event})" class = "btn-done" type = "checkbox" ${task.isDone ? 'checked' : ''}></input>
+            </div>
+            <div class = "about"> ${task.about}</div>
+            <div class = "buttons">
+            <button onclick = "deleteTask(${event})" class = "btn-delete">X</button>
+            <div class = "date-task"> ${task.datetask}</div> 
+        </div>
+    </div>    `
+}
+
+function creationDate(d = new Date()) {
+    let Y = d.getFullYear();
+    let M = ["01", "02", "03", "04", "05", "06", "07", "08", "09", "10", "11", "12"];
+    let D = d.getDate();
+    return `${D}.${M[d.getMonth()]}.${Y}`
+}
+
+const root = document.getElementById('root');
+
+const fillHtmlList = () => {
+    root.innerHTML = "";
+    tasks.forEach((item, event) => {
+        root.innerHTML += createTemplate(item, event);
+    });
+    todoItemElems = document.querySelectorAll('.todo-item');
+}
+
+function completeTask(event) {
+    tasks[event].isDone = !tasks[event].isDone
+
+    if (tasks[event].isDone) {
+        todoItemElems[event].classList.add('checked');
+    } else {
+        todoItemElems[event].classList.remove('checked');
+    }
+    updateStorage();
+    fillHtmlList();
+}
+
+const aboutTask = document.getElementById('about-task');
+
+const addBtn = document.getElementById('add-btn').addEventListener('click', () => {
```

Comment:
тут наверное переменная не нужна
```suggestion
document.getElementById('add-btn').addEventListener('click', () => {
```